test/dlc_tests/conftest.py

```diff
@@ -58,7 +58,7 @@
     "pytorch_inference_eia",
     "pytorch_inference_neuron",
     "pytorch_inference_neuronx",
-    "pytorch_training_neuron",
+    "pytorch_training_neuronx",
```

Comment:
This change would be backward incompatible. For eg., the `pytorch_training_neuron` images will not be processed by the tests anymore. With the release of new SDK, will the older Neuron images become obsolete? 

If that is the case, its fine to rename the fixture.

Reply:
Unlikely this will be needed again, training images should use the name 'neuronx'